packages/integrations/vercel/src/serverless/adapter.ts

```diff
@@ -52,12 +57,14 @@ export default function vercelServerless({
 				}
 				const outDir = getVercelOutput(config.root);
 				const viteDefine = exposeEnv(['VERCEL_ANALYTICS_ID']);
+				serverBuildDir = new URL('./dist/', config.root);
 				updateConfig({
 					outDir,
 					build: {
 						serverEntry: 'entry.mjs',
 						client: new URL('./static/', outDir),
-						server: new URL('./dist/', config.root),
+						server: serverBuildDir,
+						excludeMiddleware: true,
```

Comment:
Per our conversation, we are going to let the user toggle this themselves, right?

Reply:
Good catch! Yes, that's intention